stable/anchore-engine/templates/engine_upgrade_job.yaml

```diff
@@ -43,7 +43,7 @@ spec:
         {{- end }}
         command: ["/bin/bash", "-c"]
         {{- if .Values.anchoreGlobal.dbConfig.ssl }}
-        args: ["anchore-manager db --db-use-ssl --db-connect postgresql://${ANCHORE_DB_USER}:${ANCHORE_DB_PASSWORD}@${ANCHORE_DB_HOST}/${ANCHORE_DB_NAME}?sslmode={{- .Values.anchoreGlobal.dbConfig.sslMode -}}&sslrootcert=/home/anchore/certs/{{- .Values.anchoreGlobal.dbConfig.sslRootCertName -}} upgrade --dontask"]
+        args: ["anchore-manager db --db-use-ssl --db-connect postgresql://${ANCHORE_DB_USER}:${ANCHORE_DB_PASSWORD}@${ANCHORE_DB_HOST}/${ANCHORE_DB_NAME}?sslmode={{ .Values.anchoreGlobal.dbConfig.sslMode -}}\\&sslrootcert=/home/anchore/certs/{{ .Values.anchoreGlobal.dbConfig.sslRootCertName }} upgrade --dontask"]
```

Comment:
I think that there is only 1 whitespace chomping error here.
This should be okay:
```
{{- .Values.anchoreGlobal.dbConfig.sslMode -}}
```
However, this is incorrect:
```
{{- .Values.anchoreGlobal.dbConfig.sslRootCertName -}} 
```
it should be:
```
{{- .Values.anchoreGlobal.dbConfig.sslRootCertName }}
```

Reply:
It is not wrong but imho generally not needed or wanted when expanding in a string like this. As no other in-string interpolation in this file was written with chomping either I thought removing it completely might prevent it spreading to more places where it might cause more bugs. I can of course reduce the patch to just fixing the bug if you want though.

Reply:
@hacst the chomping was added to this because it is part of a connection string that should not include any white space in it. I would prefer that the template used what I suggested above. Thanks!

Reply:
@Btodhunter  Will do so.

To make sure we are on the same page: If the variable being interpolated has e.g. trailing whitespace in it, it will still end up in the connection string with chomping. Chomping will only do something when a dev directly puts whitespace into the template.

Reply:
@hacst thanks for the clarification. With that point, I think you're probably right that we do not need the whitespace chomping. Sorry for the back & forth.

Reply:
That's fine. Will adjust the last patch.